.github/workflows/instances.yml

```diff
@@ -128,4 +135,4 @@ jobs:
           body: This PR is updating `lemmy-instances.txt`, `instances.dart`, `AndroidManifest.xml`, `manifest.json` and `content.js` with the latest list of instances retrieved from fediverse.observer.
           branch: update-instances
           delete-branch: true
-          author: GitHub <noreply@github.com>
+          author: GitHub <noreply@github.com>
```

Comment:
Did you want to add back the newline at the end of this file? Not a big deal either way.